Explorer/Assets/Scripts/CrdtEcsBridge/JsModulesImplementation/SDKObservableEventsEngineAPIImplementation.cs

```diff
@@ -0,0 +1,44 @@
+using CRDT.Deserializer;
+using CRDT.Protocol;
+using CRDT.Protocol.Factory;
+using CRDT.Serializer;
+using CrdtEcsBridge.OutgoingMessages;
+using CrdtEcsBridge.PoolsProviders;
+using CrdtEcsBridge.UpdateGate;
+using CrdtEcsBridge.WorldSynchronizer;
+using DCL.ECS7;
+using SceneRunner.Scene.ExceptionsHandling;
+using SceneRuntime.Apis.Modules.EngineApi.SDKObservableEvents;
+using System.Collections.Generic;
+using Utility.Multithreading;
+
+namespace CrdtEcsBridge.JsModulesImplementation
+{
+    public class SDKObservableEventsEngineAPIImplementation : EngineAPIImplementation, ISDKObservableEventsEngineApi
+    {
+        public HashSet<ProcessedCRDTMessage> OutgoingCRDTMessages { get; } = new ();
```

Comment:
I don't think you can provide any unique checks by `HashSet` as `ProcessedCRDTMessage` does not provide `IEquatable` implementation and it seems to be not needed: I propose to use a simple list instead

Reply:
Makes sense, I updated that to a List and also replaced `ProcessedCRDTMessage` with just `CRDTMessage` since we don't need the container structure of the `ProcessedCRDTMessage`